server/tree-sitter-bash.info

```diff
@@ -1,2 +1,2 @@
-"https://api.github.com/repos/tree-sitter/tree-sitter-bash/git/commits/2fbd860f802802ca76a6661ce025b3a3bca2d3ed"
-tree-sitter-cli "^0.22.6"
```

Comment:
removal of the quotes should not have any impact. This is just informative for the github CI cache